data/human/human missions.txt

```diff
@@ -4965,3 +4965,164 @@ mission "Riley"
 			label end
 			`	You say your goodbyes, leaving to your ship and returning to the stars.`
 				decline
+
+mission "A dead man's path I"
+	name "Siblings to New China"
+	description "Bring these siblings to New China to hear their father's last will."
+	passengers 2
+	source
+		near "Merak" 3 6
```

Comment:
```suggestion
		near "Merak" 3 6
		not attributes "station"
```
It wouldn't work with one of my suggestions, and also it would make more sense. There is a chance that can be offered on somewhere that wouldn't work like Huginn Station.